console/src/pages/Model/ModelOverviewLayout.tsx

```diff
@@ -107,8 +133,16 @@ export default function ModelOverviewLayout({ children }: IModelLayoutProps) {
                 pattern: '/\\/files\\/?',
             },
         ]
+
+        if (servingJobs?.length) {
+            items.push({
+                title: t('Servings'),
+                path: `/projects/${projectId}/models/${modelId}/versions/${modelVersionId}/servings`,
+                pattern: '/\\/servings\\/?',
+            })
+        }
         return items
-    }, [projectId, modelId, modelVersionId, t])
+    }, [t, projectId, modelId, modelVersionId, servingJobs?.length])
```

Comment:
why use .length as dep here, servingJobs will be better ?

Reply:
It can reduce the number of recalculations
online demo: https://codesandbox.io/s/use-state-dep-8s3c3r